engines/tetraedge/te/te_font3.cpp

```diff
@@ -144,7 +145,10 @@ void TeFont3::draw(TeImage &destImage, const Common::String &str, int fontSize,
 
 	uint32 uintcol = ((uint32)col.a() << fmt.aShift) | ((uint32)(col.r()) << fmt.rShift)
 						| ((uint32)(col.g()) << fmt.gShift) | ((uint32)(col.b()) << fmt.bShift);
-	font->drawString(&destImage, str, 0, yoff, destImage.w, uintcol, talign);
+	Common::String line(str);
+	if (g_engine->getCore()->language() == "he")
+		line = convertBiDiString(str, Common::kWindows1255);
```

Comment:
I think this should be namespaced, eg `Common::convertyBiDiString`, right?

Reply:
Thank you, added namespace